<launch>
	<!-- Lidar odometry param -->
	<rosparam file="$(find lvi_sam)/config/KITTI_lidar.yaml" command="load" />
	<!-- VINS config -->
	<param name="vins_config_file" type="string" value="$(find lvi_sam)/config/KITTI_camera.yaml" />


    <!--- Robot State TF -->
    <include file="$(find lvi_sam)/launch/include/module_robot_state_publisher.launch" />
    <!--- Run Rviz-->
    <include file="$(find lvi_sam)/launch/include/module_rviz.launch" />
    <!--- SAM -->
    <include file="$(find lvi_sam)/launch/include/module_sam.launch" />
    
    <!-- scan_to_map: LOAM-style scan2map optimizer (added for KITTI runs) -->
    <node name="scan_to_map" pkg="lvi_sam" type="lvi_sam_scan_to_map" output="screen">
        <!-- If your PROJECT_NAME differs or topics remapping is needed, add <remap> entries here -->
    </node>
</launch>
